data/deep missions.txt

```diff
@@ -1952,7 +1958,7 @@ mission "Deep: Scientist Rescue 1"
 	npc save accompany
 		government Republic
 		personality escort
-		ship "Mule (Heavy)" "D.S.S. Faraday"
+		ship "Mule (Heavy)" "D.P.S. Faraday"
```

Comment:
Given that Deep Archaeology is the only place that D.P.S. ships appear, I think it's safe to assume that D.P.S. and D.S.S. are two different things; one is Deep Security (handle foreign matters and local defense) and the other is likely Deep Police (handle local matters). Think of Deep Security as being the CIA and Homeland Security put together and Deep Police as being the FBI, and given what Paris' role in these missions, that would put him in Deep Security (D.S.S.).

Reply:
fair and good point